source/parse.h

```diff
@@ -5825,8 +5867,6 @@ class parser
             error("prefix '++var' is not valid Cpp2; use postfix 'var++' instead", false);
         break; case lexeme::MinusMinus:
             error("prefix '--var' is not valid Cpp2; use postfix 'var--' instead", false);
-        break; case lexeme::Multiply:
-            error("prefix '*ptr' dereference is not valid Cpp2; use postfix 'ptr*' instead", false);
```

Comment:
This diagnostic should be restored when it doesn't happen at the start of a function argument.